### DISCUSSION TITLE
Receiving subscription notification although value did not change

### DISCUSSION THREAD

**Discussion by ultra7677** [eclipse-milo/milo#1033] [General]
Hi,

We are facing this issue in our production environment, can you please give us some hints on this?

The client subscribes to around 3500 nodes at the same time, using the default sampling interval of 1000ms

The server sampling interval is set to 500ms, the publishing interval is 500ms, and the maximum MonitoredItem is set to 7000.

**Comment by kevinherron** [comment]
Are you using Milo for the client or the server? Are you sure the notification is not just a keep-alive message?

**Reply by ultra7677** [reply]
Thanks for the quick reply! 

I am using Milo for the client side, and the OPC UA server is a Siemens TIA 1500 PLC,  I am pretty sure the notification is not a keep-alive message. It should be a PLC-related issue, however, I have no clue how to dig into this.

The following log shows the detail, we are subscribing to one node, and it keeps sending "false" with different timestamps. This issue occurs last day and after around 16 hours, everything was back to normal.


![bRSrB4JF8x](https://user-images.githubusercontent.com/11144269/186831794-c738e8e6-9386-42c8-9357-561f3e74f063.jpg)

![image](https://user-images.githubusercontent.com/11144269/186832101-a5ff411d-2b22-47d0-859e-58df56b55a67.png)

**Reply by kevinherron** [reply]
Okay, assuming the redacted NodeId is actually the same, I can only think of a few reasons for this:
1. the MontioredItem was created with `DataChangeTrigger.StatusValueTimestamp`
2. the ResendData method is being invoked for some reason
3. bug in the server

**Reply by ultra7677** [reply]
Thanks for the hint, we are trying to change the MonitoredItem to do not use timestamp

**Reply by ultra7677** [reply]
@kevinherron Hi Kevin, we figure out the reason, it's a PLC issue, we use Siemens-1515 which recommends use 2000 MonitoredItem, however, we created 2500 MonitoredItems and caused the PLC overload, we fixed this issue by upgrade the PLC
